Install-DVWA.sh

```diff
@@ -0,0 +1,174 @@
+mysql -u root -p$mysql_root_password -e "FLUSH PRIVILEGES;"
+echo
+
+# Success message
+success_message=$(get_language_message "\e[92mMySQL commands executed successfully.\e[0m" "\e[92mComandos MySQL ejecutados correctamente.\e[0m")
+echo -e "$success_message"
+
+sleep 2
+
+# Coping DVWA folder to /var/www/html
+dvwa_config_message=$(get_language_message "\e[96mConfiguring DVWA...\e[0m" "\e[96mConfigurando DVWA...\e[0m")
+echo -e "$dvwa_config_message"
+cp /var/www/html/DVWA/config/config.inc.php.dist /var/www/html/DVWA/config/config.inc.php
+sleep 2
+
+# Assign the right permissions to DVWA
+permissions_config_message=$(get_language_message "\e[96mConfiguring permissions...\e[0m" "\e[96mConfigurando permisos...\e[0m")
+echo -e "$permissions_config_message"
+chown -R www-data:www-data /var/www/html/DVWA
+chmod -R 755 /var/www/html/DVWA
+sleep 2
+
+php_config_message=$(get_language_message "\e[96mConfiguring PHP...\e[0m" "\e[96mConfigurando PHP...\e[0m")
+echo -e "$php_config_message"
+# Trying to find the php.ini file in the Apache folder
+php_config_file_apache="/etc/php/$(php -r 'echo PHP_MAJOR_VERSION . "." . PHP_MINOR_VERSION;')/apache2/php.ini"
+
+# Trying to find the php.ini file in the FPM folder
+php_config_file_fpm="/etc/php/$(php -r 'echo PHP_MAJOR_VERSION . "." . PHP_MINOR_VERSION;')/fpm/php.ini"
+
+# Check if the php.ini file exists in the Apache folder and use it if it is present.
+if [ -f "$php_config_file_apache" ]; then
+    php_config_file="$php_config_file_apache"
+    sed -i 's/^\(allow_url_include =\).*/\1 on/' $php_config_file
+    sed -i 's/^\(allow_url_fopen =\).*/\1 on/' $php_config_file
+    sed -i 's/^\(display_errors =\).*/\1 on/' $php_config_file
+    sed -i 's/^\(display_startup_errors =\).*/\1 on/' $php_config_file
+# Check if the php.ini file exists in the FPM folder and use it if it is present.
+elif [ -f "$php_config_file_fpm" ]; then
+    php_config_file="$php_config_file_fpm"
+    sed -i 's/^\(allow_url_include =\).*/\1 on/' $php_config_file
+    sed -i 's/^\(allow_url_fopen =\).*/\1 on/' $php_config_file
+    sed -i 's/^\(display_errors =\).*/\1 on/' $php_config_file
+    sed -i 's/^\(display_startup_errors =\).*/\1 on/' $php_config_file
+else
+    # Warning message if not found in any of the folders
+    php_file_message=$(get_language_message "\e[91mWarning: PHP configuration file not found in Apache or FPM folders.\e[0m" "\e[91mAdvertencia: No se encuentra el fichero de configuración PHP en las carpetas de Apache o FPM.\e[0m")
+    echo -e "$php_file_message"
+fi
+sleep 2
+
+# Apache restart
+apache_restart_message=$(get_language_message "\e[96mRestarting Apache...\e[0m" "\e[96mReiniciando Apache...\e[0m")
+echo -e "$apache_restart_message"
+systemctl restart apache2
+sleep 2
+
+# Show to user the credentials for first use
+credentials_message=$(get_language_message "\e[92mUsername and password for the first use:\e[0m" "\e[92mUsuario y contraseña para el primer uso:\e[0m")
```

Comment:
What do you mean first use? These are the database credentials.

Reply:
I mean that you need to use the database credentials to run it for the first time, after the setup will be admin, password

Reply:
No you don't. The user only ever uses the admin/password credentials, the database creds only go in the config file.

Reply:
It doesn't matter I just realized😅